src/box/lua/config/applier/roles.lua

```diff
@@ -44,7 +45,7 @@ local call_on_event_callbacks = locked(function(config, key, value)
             log.verbose(('roles.on_event: calling callback for role ' ..
                          '"%s"'):format(role_name))
             local ok, err = pcall(role.on_event,
-                                  config, key, value)
+                                  config[role_name], key, value)
```

Comment:
Should we rename the argument here from `config` to `roles_cfg` as well (to use one term for one object)?

Reply:
done!